### PR TITLE
Add Ecko (Mastodon fork)

### PR DESCRIPTION
**Pull request by weex**
Repo doesn't exist at this instant but working to transfer it from from weex/ecko_ynh to YunoHost-Apps.

### CODE REVIEW

File: apps.json [hunk 1]
```diff
@@ -702,6 +702,16 @@
         ],
         "url": "https://github.com/YunoHost-Apps/easyappointments_ynh"
     },
+    "ecko": {
+        "branch": "master",
+        "category": "social_media",
+        "level": 7,
```
Comment (yalh76) [inline apps.json:708]:
```suggestion
```

level will be calculated automatically

File: apps.json [hunk 2]
```diff
@@ -702,6 +702,16 @@
         ],
         "url": "https://github.com/YunoHost-Apps/easyappointments_ynh"
     },
+    "ecko": {
+        "branch": "master",
```
Comment (yalh76) [inline apps.json:706]:
```suggestion
```

Branch is not required

**Review by yalh76 — CHANGES_REQUESTED** [review]

**Comment by weex** [comment]
Updated to remove those two lines.

**Review by tituspijean — APPROVED** [review]
Thank you for your work, and welcome!

**Review by yalh76 — APPROVED** [review]

**Pull request merged**